### PR TITLE
Add project README

### PR DESCRIPTION
**Pull request by jianzzha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Netgauge Project
+
+The netgauge project develops components that are used for telco edge node user space networking performance test and telemetry data collection. The kernel networking telemetry collection is not in the scope of this project, nor is the kernel networking performance test tooling.
```
Comment (dkosteck) [inline README.md]: "user space performance testing" maybe?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+The user space networking performance test components include:
+* [A RFC2544 traffic generator container](rfc2544/README.md) that supports REST API for remote control and data collection
```
Comment (dkosteck) [inline README.md]: APIs

File: README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+* [A testpmd wrapper container](testpmd/README.md) that supports REST API for remote control and data collection
```
Comment (dkosteck) [inline README.md]: APIs

**Review by dkosteck — CHANGES_REQUESTED** [review]
Small formatting changes

**Review by akiselev1 — COMMENTED** [review]
Should we mention somewhere that these tools development and verification was based on RHEL and Intel NICs? There may be some difference on other untested platforms.

File: README.md (hunk 1 above)
Comment (jianzzha) [reply README.md]: We only care about the networking performance tests and telemetry data, not other performance tests - the performance testing has a lot of other aspects

File: README.md (hunk 2 above)
Comment (jianzzha) [reply README.md]: ack

File: README.md (hunk 3 above)
Comment (jianzzha) [reply README.md]: ack

**Comment by jianzzha** [comment]
That info should not be under the root readme. As the trafficgen and testpmd will not be the only components for this repo.

The RHEL should not be a factor here, container should work across different linux distro.

As to the NIC, we can add some pointers to the vendor's documentation. For example,  to TREX's document which lists which NICs are supported. We will do that in a separate PR

**Review by dkosteck — APPROVED** [review]
Looks good

**Review by akiselev1 — APPROVED** [review]

**Pull request merged**